src/app/chartTypes.ts

```diff
@@ -18,6 +18,7 @@ const chartGroups = [
           'showLegend',
           'legendTitle',
           'legendPosition',
+          'dataLabelPosition',
```

Comment:
Show data label is supported for 8 charts currently - standard, stacked, grouped vertical bar chart, standard stacked, grouped horizontal bar chart and negative stacked horizontal and vertical bar charts. However, data label inside bar is only supported for standard vertical and horizontal bar charts. Is it possible to add the feature to the rest of the charts that support data label?